modules/uhttp/client/client_test.go

```diff
@@ -32,10 +33,11 @@ import (
 	"golang.org/x/net/context"
 )
 
-var _defaultUHTTPClient = NewClient(*_defaultHTTPClient)
+var _defaultHTTPClient = &http.Client{Timeout: 2 * time.Second}
+var _defaultUHTTPClient = New(*_defaultHTTPClient)
 
-func TestNewClient(t *testing.T) {
-	uhttpClient := NewClient(*_defaultHTTPClient)
+func TestNew(t *testing.T) {
+	uhttpClient := New(*_defaultHTTPClient)
```

Comment:
Read a bit about this and pointed embed makes sense in this case based on http://stackoverflow.com/questions/28501976/embedding-in-go-with-pointer-or-with-value. Changed.